lib/node_modules/@stdlib/stats/base/dists/geometric/pmf/src/main.c

```diff
@@ -0,0 +1,70 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/stats/base/dists/geometric/pmf.h"
+#include "stdlib/math/base/assert/is_nonnegative_integer.h"
+#include "stdlib/math/base/assert/is_nan.h"
+#include "stdlib/math/base/special/pow.h"
+#include <math.h>
+
+/**
+* Evaluates the probability mass function (PMF) for a geometric distribution with success probability `p` at a value `x`.
+*
+* @param {number} x - input value
```

Comment:
Shouldn't have JSDoc comments in C files. No type annotations and descriptions should be vertically aligned.